mlflow/fastai.py

```diff
@@ -398,6 +400,7 @@ def autolog(log_models=True, disable=False, exclusive=False):  # pylint: disable
     :param exclusive: If ``True``, autologged content is not logged to user-created fluent runs.
                       If ``False``, autologged content is logged to the active fluent run,
                       which may be user-created.
+    :param disable_for_untested_versions: Disable for untested versions
```

Comment:
Can we be a bit more specific with the docs here? E.g., "If True, disable autologging for versions of fastai that have not been tested against this version of the MLflow client or are incompatible." (Feel free to modify / refine the phrasing here)